### PR TITLE
CI: Rust CI path filter

### PR DESCRIPTION
**Pull request by michalsanger**
Run CI for Rust projects only when there are changed files in the pull request

### CODE REVIEW

**Comment by michalsanger** [comment]
Not sure this will work because skipped checks are marked as required...

**Comment by mrtnzlml** [comment]
> Not sure this will work because skipped checks are marked as required...

@michalsanger I am afraid it won't work. However, you can use [dorny/paths-filter](https://github.com/dorny/paths-filter) to workaround this issue which should work fine. We use it in Dockerfile builder: https://github.com/adeira/universe/blob/00413dbb8c38fe48e7476a48d51486121adb8e5b/.github/workflows/dockerfile-builder.yml#L37-L42

**Comment by michalsanger** [comment]
`dorny/paths-filter` looks good. Just to be sure, could you try to approve this PR?

**Review by mrtnzlml — APPROVED** [review]

**Comment by michalsanger** [comment]
WTF kodiak! :-)

File: .github/workflows/continuous-integration-rust.yml [hunk 1]
```diff
@@ -24,8 +24,17 @@ jobs:
     steps:
       - uses: actions/checkout@v2.3.4
 
+      - uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          filters: |
+            src:
+              - 'src/signedsource-rs/**'
+              - 'src/abacus/**'
```
Comment (mrtnzlml) [inline .github/workflows/continuous-integration-rust.yml:33]: One slight improvement for the future would be to use the test matrix above.

**Pull request merged**